charts/descheduler/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+name: descheduler
+version: 1.0.0
+appVersion: 0.18.0
```

Comment:
Should this be `v0.18.0` to match the git tag for descheduler?

Reply:
The convention in Helm charts is not to add the _v_ prefix to versions (see https://hub.helm.sh/).

Reply:
Looks good.